packages/build/src/core/constants.js

```diff
@@ -73,6 +78,22 @@ const getConstants = async function ({
   return constantsA
 }
 
+// Temporary warning until we launch this feature in February 2021
+// TODO: remove once the feature is launched
+const validateNetlifyDir = async function (logs, buildDir) {
+  if (await pathExists(`${buildDir}/${DEFAULT_FUNCTIONS_SRC}`)) {
```

Comment:
Forward slashes work on Windows (no need for `path.join()`) with Node.js core `fs` API (which `path-exists` uses).